File: PHP/Program/Product.php
<?php

class Product{
    private $id;
    private $nama;
    private $brand;
    private $harga;

    function __construct($id = '', $nama = "", $brand = "", $harga = "") {
        $this->id = $id;
        $this->nama = $nama;
        $this->brand = $brand;
        $this->harga = $harga;
    }

    

    public function getId() {
      return $this->id;
    }
    public function setId($id) {
      $this->id = $id;
    }

    public function getNama() {
      return $this->nama;
    }
    public function setNama($nama) {
      $this->nama = $nama;
    }

    public function getBrand() {
      return $this->brand;
    }
    public function setBrand($brand) {
      $this->brand = $brand;
    }

    public function getHarga() {
      return $this->harga;
    }
    public function setHarga($harga) {
      $this->harga = $harga;
    }
}







?>

File: CPP/Program/Product.cpp
#include <iostream>
#include <string>

using namespace std;

class Product{
    private:
        int id;
        string nama;
        string brand;
        int harga;
    public:

        Product(){
            this->id = '0';
            this->nama = "";
            this->brand = "";
            this->harga = '0';
        }

        Product(int id, string nama, string brand, int harga){
            this->id = id;
            this->nama = nama;
            this->brand = brand;
            this->harga = harga;
        }


         // Getter
        int getId() { 
            return id; 
        }
        string getNama() { 
            return nama; 
        }
        string getBrand() { 
            return brand; 
        }
        int getHarga() { 
            return harga; 
        }

        // Setter
        void setId(int id) { 
            this->id = id; 
        }
        void setNama(string nama) { 
            this->nama = nama; 
        }
        void setBrand(string brand) { 
            this->brand = brand; 
        }
        void setHarga(int harga) { 
            this->harga = harga; 
        }

        ~Product(){
            
        }
};